src/metrics/mendel_metrics.nit

```diff
@@ -37,10 +40,10 @@
 # * pure replacers that never call to super in its redefinitions
 # * replacers that have less redefinitions that call super than not calling it
 #
-# For more details see
-#  Mendel: A Model, Metrics and Rules to Understan Class Hierarchies
-#  S. Denier and Y. Gueheneuc
-#  in Proceedings of the 16th IEEE International Conference on Program Comprehension (OCPC'08)
+# For more details see S. Denier and Y. Gueheneuc, “Mendel: A Model, Metrics and
+# Rules to Understand Class Hierarchies,” in *Proceedings of the 16th IEEE
+# International Conference on Program Comprehension* (OCPC'08), Amsterdam, 2008,
+# pp. 143-152. doi: 10.1109/ICPC.2008.37
```

Comment:
I think I will at least fix the spelling mistakes and add some formatting for the web output (currently unligible). Is it OK for you?